addons/main/stringtable.xml

```diff
@@ -14,6 +14,7 @@
         </Key>
         <Key ID="STR_CBA_URL">
             <English>https://www.github.com/CBATeam/CBA_A3</English>
+            <German>https://www.github.com/CBATeam/CBA_A3</German>
```

Comment:
Why not just simply use a single <Original> entry here as every entry is literally the same

Reply:
This is a string so it can be changed if needed everywhere at once, while not having to use a macro which would have character escaping issues. It is translated I guess mainly so Jonpas' translation counter bot says 100% eventually.

Reply:
Flummi's bot. ;)
But yes.